src/uploadscreens.py

```diff
@@ -362,7 +362,7 @@ def extract_numeric_suffix(filename):
     default_pool_size = int(meta.get('task_limit', os.cpu_count()))
     host_limits = {"oeimg": 6, "ptscreens": 1, "lensdump": 1}
     pool_size = host_limits.get(img_host, default_pool_size)
-    max_workers = min(len(upload_tasks), pool_size)
+    max_workers = max(1, min(len(upload_tasks), pool_size))
```

Comment:
If upload_tasks = 0, this indicates that no images will be uploaded, and in my view it should hard error, rather than upload with no images in the description.

Reply:
I agree.  But I was able to confirm that there were in fact images to upload (as evidenced by images being uploaded when I ran this fix locally).  I'll look into it and also found another bug that occurs on a subsequent run so I will convert this back to draft for now.

Reply:
It was every attempt, yes.

Reply:
Seems like something in the `default_pool_size` then. If you have task_limit commented out in config, then it must be something with the cpu_count.